setup_utils.py

```diff
@@ -0,0 +1,149 @@
+import ast
+import os
+import re
+import subprocess
+
+
+_R_RC = re.compile("rc(\d+)$")
+
+
+def parse_version(path):
+    with open(path) as fp:
+        return _AssignmentParser().parse(fp.read())["version"]
+
+
+def write_version_py(filename, major, minor, micro, is_released,
+                     previous_version=None):
```

Comment:
Right now, the build number is calculated from the first commit. We're already at 704 for  enthought/satsolver, 994 for enthought/jaguar and 3700 for enthought/enstaller (and ~17000 for the buildsystem :) )